watertap/property_models/anaerobic_digestion/adm1_reactions.py

```diff
@@ -0,0 +1,904 @@
+            doc="Lipids from composites",
+        )
+        self.N_xc = pyo.Var(
+            initialize=0.002,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Nitrogen content of composites",
+        )
+        self.N_I = pyo.Var(
+            initialize=0.002,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Nitrogen content of inerts",
+        )
+        self.f_fa_li = pyo.Var(
+            initialize=0.95,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Fatty acids from lipids",
+        )
+        self.f_h2_su = pyo.Var(
+            initialize=0.19,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Hydrogen from sugars",
+        )
+        self.f_bu_su = pyo.Var(
+            initialize=0.13,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Butyrate from sugars",
+        )
+        self.f_pro_su = pyo.Var(
+            initialize=0.27,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Propionate from sugars",
+        )
+        self.f_ac_su = pyo.Var(
+            initialize=0.41,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Acetate from sugars",
+        )
+        self.f_h2_aa = pyo.Var(
+            initialize=0.06,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Hydrogen from amino acids",
+        )
+        self.N_aa = pyo.Var(
+            initialize=0.007,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Nitrogen in amino acids and proteins",
+        )
+        self.f_va_aa = pyo.Var(
+            initialize=0.23,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Valerate from amino acids",
+        )
+        self.f_bu_aa = pyo.Var(
+            initialize=0.26,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Butyrate from amino acids",
+        )
+        self.f_pro_aa = pyo.Var(
+            initialize=0.05,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Propionate from amino acids",
+        )
+        self.f_ac_aa = pyo.Var(
+            initialize=0.40,
+            units=pyo.units.dimensionless,
+            domain=pyo.PositiveReals,
+            doc="Acetate from amino acids",
+        )
+        # Kinetic Parameters
+
+        # Reaction Stoichiometry
+        # This is the stoichiometric part of the Peterson matrix in dict form.
+        # Note that reaction stoichiometry is on a mass basis.
+        # See Table 3.1 in Batstone et al., 2002.
+        mw_n = 14 * pyo.units.kg / pyo.units.kmol
```

Comment:
This was leftover from ASM1. I didn't delete, just in case it ends up being necessary for the next set of things that I'm adding. If unneeded after that, will remove.

Reply:
OK. That makes sense, it's under developing, I'll focus on double checking those parameter values and expressions